app/static/src/app/store/graphs/state.ts

```diff
@@ -10,9 +10,16 @@ export interface GraphConfig {
     id: string; // We need to keep a persistent id to identify configs in vue when a graph is deleted from the array
     selectedVariables: string[];
     unselectedVariables: string[]; // We keep track of unselected variables too so we can retain on model update
+    settings: GraphSettings;
 }
 
 export interface GraphsState {
     config: GraphConfig[];
-    settings: GraphSettings; // TODO: this will be replaced by per-graph settings in mrc-5442
+    fitGraphSettings: GraphSettings; // For Fit apps, the Fit tab graph needs to have its own settings
 }
+
+export const defaultGraphSettings = (): GraphSettings => ({
+    logScaleYAxis: false,
+    lockYAxis: false,
+    yAxisRange: [0, 0]
+});
```

Comment:
For newcomers to the codebase, it would be extremely helpful to have commentary in this file that explains the difference between "graph settings", "graph config", and "graphs state" -- the names are roughly synonymous.

Reply:
Excellent point, will add some comments